core/cache/model.go

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 const modelConfigChange = "model-config-change"
+const modelMachineChange = "model-machine-change"
```

Comment:
Maybe this should be modelMachineAddRemove, since it doesn't cover changes to existing machines?